app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/confirmation_of_submission.scala.html

```diff
@@ -84,6 +84,8 @@
   }
 }
 
+@isPeriodOfAdmin = @{userKey.contains(".personalRepInPeriodOfAdmin")}
```

Comment:
`representativeType.contains(PersonalRepresentativeInPeriodOfAdmin)` is a more reliable way of determining this flag - strings can be fickle things

Reply:
that's done